Resources/Prototypes/Entities/Objects/Misc/handcuffs.yml

```diff
@@ -117,3 +117,25 @@
     sprite: Objects/Misc/cablecuffs.rsi
     state: cuff-broken
     color: forestgreen
+
+- type: entity
+  parent: Handcuffs
+  id: ClothingOuterStraightjacket
+  name: straitjacket
+  description: You should probably be wearing this.
+  components:
+  - type: Item
+    size: 10
+  - type: Handcuff
+    cuffedRSI: Clothing/OuterClothing/Misc/straight_jacket.rsi
+    breakoutTime: 60
+    damageOnResist:
+      types:
+        Blunt: 0
+    cuffTime: 6.5
+    uncuffTime: 5.5
+    stunBonus: 4
+    bodyIconState: body-overlay
```

Comment:
Isnt this field unnecesary? Also make cuffing/uncuffing take 15 secs and make it size 20

Reply:
The straitjacket functions as an overlay, i think it is needed. Will change all the timings and size to these numbers.

Reply:
what i meant that the value of bodyIconState already defaults to body-overlay

Reply:
do test tho

Reply:
I removed it, yea works fine.